contrib/src/main/java/com/datatorrent/contrib/kafka/AbstractKafkaInputOperator.java

```diff
@@ -203,6 +204,14 @@ public AbstractKafkaInputOperator()
    */
   protected abstract void emitTuple(Message message);
```

Comment:
I don't see a backward compatibility issue, it's a new method. And yes, the same change needs to be made on the 0.9 operator also.